rmgpy/rmg/main.py

```diff
@@ -552,10 +568,16 @@ def execute(self, **kwargs):
                     atol = self.simulatorSettingsList[0].atol,
                     rtol = self.simulatorSettingsList[0].rtol,
                     filterReactions=True,
+                    conditions = self.RMG_Memories[index].getCond()
                 )
                 self.updateReactionThresholdAndReactFlags(
                     rxnSysUnimolecularThreshold=reactionSystem.unimolecularThreshold, 
                     rxnSysBimolecularThreshold=reactionSystem.bimolecularThreshold)
+        else:
+            for index, reactionSystem in enumerate(self.reactionSystems):
+                self.RMG_Memories.append(RMG_Memory(reactionSystem))
+                self.RMG_Memories[index].generateCond()
+                logging.info('conditions choosen for reactor {0} were {1}'.format(index,self.RMG_Memories[index].getCond()))
```

Comment:
What's the difference between this an `logConditions()`?

Reply:
fixed